types/fhir-kit-client/tsconfig.json

```diff
@@ -3,11 +3,12 @@
         "module": "commonjs",
         "target": "es6",
         "lib": [
-            "es6"
+            "es6",
+            "dom"
         ],
         "esModuleInterop": true,
         "allowSyntheticDefaultImports": true,
-        "noImplicitAny": true,
+        "noImplicitAny": false,
```

Comment:
please keep this as `true`. Otherwise you may introduce `undefined` errors for package users without realising it.

Reply:
Hi @sandersn  -- When I try this I see this error but can't figure out what is causing it? Any pointers on how to fix this?

```
> definitely-typed@0.0.3 test /Users/bkaney/code/fkc/DefinitelyTyped
> dtslint types "fhir-kit-client"

Error: Errors in typescript@4.2 for external dependencies:
../request/index.d.ts(24,27): error TS7016: Could not find a declaration file for module 'form-data'. '/Users/bkaney/code/fkc/DefinitelyTyped/node_modules/form-data/lib/form_data.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/form-data` if it exists or add a new declaration (.d.ts) file containing `declare module 'form-data';`

    at /Users/bkaney/code/fkc/DefinitelyTyped/node_modules/dtslint/bin/index.js:206:19
    at Generator.next (<anonymous>)
    at fulfilled (/Users/bkaney/code/fkc/DefinitelyTyped/node_modules/dtslint/bin/index.js:6:58)
npm ERR! Test failed.  See above for more details.
```

Reply:
Try `npm run test-all`, since it looks like `request` is missing an `npm install`.